velox/exec/Window.cpp

```diff
@@ -99,9 +99,10 @@ Window::WindowFrame Window::createWindowFrame(
       auto constant =
           std::dynamic_pointer_cast<const core::ConstantTypedExpr>(frame)
               ->value();
-      VELOX_CHECK(!constant.isNull(), "k in frame bounds must not be null");
+      VELOX_CHECK(!constant.isNull(), "Window frame offset must not be null");
```

Comment:
Presto java seems to always use the error message "Window frame offset value must not be negative or null".

Lets be consistent with that.

Reply:
Thanks for sharing, I see that this error message is used with assertQueryFails in frame offset tests. However, the function ```private long getFrameValue(int channel, String type)``` has different error messages which the current change is consistent with (except for the frame type "starting" and "ending").

I ran the following query on Presto java and the error message is consistent with the Velox error for negative values:
```
presto:tpch> SELECT orderstatus, sum(orderkey) OVER (PARTITION BY orderstatus ROWS BETWEEN mod(custkey, 2) - 1 PRECEDING AND custkey / 5 FOLLOWING) from orders limit 5;

Query 20230705_224922_00004_3ngpr, FAILED, 1 node
Splits: 55 total, 6 done (10.91%)
[Latency: client-side: 0:01, server-side: 0:01] [15K rows, 377KB] [12.4K rows/s, 313KB/s]

Query 20230705_224922_00004_3ngpr failed: Window frame -1 offset must not be negative
```
For NULL offsets, the "starting" and "ending" types need to be passed for including in the error message so it is omitted.

I see the ```@Language("RegExp")``` annotation is used to account for error messages containing both null and negative with ```or```. Please let me know if the current error messages are fine or should be modified.

Reply:
That's alright then. Thanks for confirming.